zabbix_metrics_watcher.py

```diff
@@ -214,14 +216,32 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                if list(builds_in_new) == []:
+                    builds_in_new_time = datetime.datetime.now()
+                else:
+                    timediff = int((datetime.datetime.now() - builds_in_new_time).total_seconds())
+                    logger.info("%s build(s) are in New for %s sec", len(builds_in_new), timediff)
+                    _send_zabbix_message(zabbix_host, osbs_master, "new_duration", timediff)
```

Comment:
If there is a continuous stream of builds, each one staying in New until the next one is submitted but then immediately changing to Processing, this will trigger a Zabbix message.

I think you want to track when each build was first seen in New, and alert based on the maximum elapsed time since then.